plugins/hls-class-plugin/src/Ide/Plugin/Class.hs

```diff
@@ -234,18 +247,38 @@ codeAction state plId (CodeActionParams _ _ docId _ context) = liftIO $ fmap (fr
           _ -> panic "Ide.Plugin.Class.findClassFromIdentifier"
     findClassFromIdentifier _ (Left _) = panic "Ide.Plugin.Class.findClassIdentifier"
 
+    findImplementedMethods :: NormalizedFilePath -> Range -> MaybeT IO [T.Text]
+    findImplementedMethods docPath range = do
+        (HAR {hieAst = hf}, pmap) <- MaybeT . runAction "classplugin" state $ useWithStale GetHieAst docPath
+        pure
+            $ concat
+            $ pointCommand hf (fromJust (fromCurrentRange pmap range) ^. J.start & J.character -~ 1)
```

Comment:
It would be great if we could reduce the partial function usage.

Reply:
especially since you're already returning maybes and lists, so you have multiple ways of dealing with missing things!

Reply:
Um, I am not sure which function is partial here, Can you please help me with that?

Reply:
`fromJust` will crash the program if the argument is `Nothing`!

Reply:
Oh right, i copy-pasted it from above and didn't even see :see_no_evil:

Reply:
Fixed now.